pkg/datapath/linux/node.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/vishvananda/netlink"
 	"golang.org/x/sys/unix"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
nit: i don't like seeing this dependency from `pkg/datapath` to k8s just for reusing their set type. But maybe this is just me :)

Reply:
I personally don't have a strong opinion here; however, the `sets` dependency is already used in quite a few `pkg/datapath` files:

```
g grep 'k8s.io/apimachinery/pkg/util/sets' -- pkg/datapath/ ':!*test.go'
pkg/datapath/iptables/ipset/ipset.go:        "k8s.io/apimachinery/pkg/util/sets"
pkg/datapath/iptables/ipset/ops.go:        "k8s.io/apimachinery/pkg/util/sets"
pkg/datapath/iptables/reconciler.go:        "k8s.io/apimachinery/pkg/util/sets"
pkg/datapath/linux/devices_controller.go:        "k8s.io/apimachinery/pkg/util/sets"
pkg/datapath/linux/node.go:        "k8s.io/apimachinery/pkg/util/sets"
pkg/datapath/linux/node_ids.go:        "k8s.io/apimachinery/pkg/util/sets"
pkg/datapath/tables/node_address.go:        "k8s.io/apimachinery/pkg/util/sets"
```